src/test/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/HearingRepositoryFacadeTest.java

```diff
@@ -286,6 +289,7 @@ void findLastModifiedByHearingDay() {
 
     @Test
     void whenSave_thenSaveHearing_Offender_AndDefendant() {
+        when(offenderRepositoryFacade.updateOffenderIfItExists(OFFENDER)).thenReturn(OFFENDER);
         when(offenderRepository.findByCrn(CRN)).thenReturn(Optional.of(OFFENDER.withProbationStatus(NOT_SENTENCED)))
                                                 // This must return 2 separate but identical objects on both calls as they
```

Comment:
I think this comment is no longer true as this is done in the mocked out OffenderFacade, you could remove it (maybe it needs to move to the OffenderFacade test) and one of the `thenReturns`